### PR TITLE
i10 Support to settings module

### PR DESCRIPTION
**Pull request by kumarpalsinh25**

### CODE REVIEW

File: app/macos/Podfile.lock [hunk 1]
```diff
@@ -156,4 +162,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: ce7dbe26c78bfc7ba46736094c1e2d25982870fa
 
-COCOAPODS: 1.14.3
\ No newline at end of file
+COCOAPODS: 1.14.2
```
Comment (gtalha07) [inline app/macos/Podfile.lock:165]: This keeps changing, we should keep it in ```.gitignore``` :)

File: app/pubspec.lock [hunk 2]
```diff
@@ -1293,6 +1293,14 @@ packages:
       url: "https://pub.dev"
     source: hosted
     version: "3.16.7"
+  multiavatar:
```
Comment (gtalha07) [inline app/pubspec.lock:1296]: Same here, this changes aren't related to actual PR.

**Review by gtalha07 — APPROVED** [review]
Looks good, just keep an eye out on linter complains ;)

File: app/pubspec.lock (hunk 2 above)
Comment (gnunicorn) [reply app/pubspec.lock]: but the pubspec.lock needs to go into the repo or it won't properly resolve

**Pull request merged**